code/modules/examine/examine.dm

```diff
@@ -41,6 +41,8 @@
 /client/var/description_holders[0]
 
 /client/proc/update_description_holders(atom/A, update_antag_info=0)
+	if(!A)
```

Comment:
This should arguably also be resolved at the call site. Since the entire point of this proc is to work on that input atom.

Reply:
A could be deleted at any given point.

Reply:
```
runtime error:
[07:23:51]Cannot execute null.get description info().
[07:23:51]proc name: update description holders (/client/proc/update_description_holders)
[07:23:51]  source file: examine.dm,44
[07:23:51]  usr: Quinndolyn Jerome (/mob/living/carbon/human)
[07:23:51]  src: Qu1nnd0lyn (/client)
[07:23:51]  usr.loc: the floor (179,156,4) (/turf/simulated/floor/tiled)
[07:23:51]  call stack:
[07:23:51]Qu1nnd0lyn (/client): update description holders(null, 0)
[07:23:51]Quinndolyn Jerome (/mob/living/carbon/human): Examine(null)
[07:23:51]Quinndolyn Jerome (/mob/living/carbon/human): Topic("src=\[0x300004a];lookitem=\[0x...", /list (/list))
[07:23:51]Quinndolyn Jerome (/mob/living/carbon/human): Topic("src=\[0x300004a];lookitem=\[0x...", /list (/list))
[07:23:51]Qu1nnd0lyn (/client): Topic("src=\[0x300004a];lookitem=\[0x...", /list (/list), Quinndolyn Jerome (/mob/living/carbon/human))
[07:23:51]Qu1nnd0lyn (/client): Topic("src=\[0x300004a];lookitem=\[0x...", /list (/list), Quinndolyn Jerome (/mob/living/carbon/human))
```

Reply:
`Examine(null)`

Ye, this is better fixed in `Examine()` or, even better, fix it in the `Topic()` call.